cookbook/gravmag_transform_tilt.py

```diff
@@ -0,0 +1,32 @@
+"""
+GravMag: Calculate the tilt angle of a total field anomaly
+"""
+from fatiando import mesher, gridder, utils
+from fatiando.gravmag import prism, transform
+from fatiando.vis import mpl
+from fatiando.mesher import Prism
+
+
+inc, dec = -90, 0
+mag = utils.ang2vec(40, inc, dec)
+model = [Prism(-1000, 1000, -200, 200, 0, 1000, {'magnetization': mag})]
+area = [-5000, 5000, -5000, 5000]
+shape = (200, 200)
+x, y, z = gridder.regular(area, shape, z=-150)
+data = prism.tf(x, y, z, model, inc, dec)
+tilt_angle = transform.tilt(x, y, data, shape)
+
+mpl.figure()
```

Comment:
Please avoid the `mpl` module as a replacement for `matplotlib.pyplot`. This will be removed in the future. I really need to write this down somewhere or deprecate it already.